### DISCUSSION TITLE
StructuredGrid on a 3D surface

### DISCUSSION THREAD

**Discussion by 3desinc** [pyvista/pyvista#5159] [Q&A]
I am trying to generate a StructuredGrid on the surface of a 3D model like the Stanford bunny (https://en.m.wikipedia.org/wiki/File:Stanford_Bunny.stl). Somewhat similar to CylinderStructured but on a custom mesh. Is it possible?

**Comment by MatthewFlamm** [comment]
Sure but there are requirements. The requirement is that the cells are either Hexahedron or Quadrilaterals, the points can be in arbitrary positions, but the cells they define must have i-j-k ordering.  Also the cells must share points with neighbors.

**Reply by 3desinc** [reply]
All I need is quadrilaterals. Is there an example of such a use case? I am looking at this example https://docs.pyvista.org/version/stable/examples/00-load/create-structured-surface.html and specifically "struct = examples.load_structured()". I don't see this in pyvista examples library in GitHub, any thoughts?
